Misc/NEWS.d/next/Tests/2019-06-12-11-02-06.bpo-35545.s63eHE.rst

```diff
@@ -0,0 +1,4 @@
+Skip `test_asyncio.test_create_connection_ipv6_scope` on AIX 
+because "getaddrinfo()" behaves different on AIX
+
+Patch by M. Felt
```

Comment:
Honestly, it's not really worth it to document when a test is skipped on a specific platform. I suggest to remove the NEWS entry. I added the "skipped news" label.